### PR TITLE
Added Identity function

### PR DESCRIPTION
**Pull request by VMRuiz**
Added Identity Method to be able to return fixed values. 

Is it quite common that sometimes with just need to return a fixed value.

### CODE REVIEW

**Comment by renancunha** [comment]
Great feature :smiley: @VMRuiz! During the schema conversion tests, we needed a feat like this one to fill a specific field with a fixed value.

I have just one consideration: maybe should we stick to the idea of always returns the iterable (with the result), instead of returning the result itself? By doing this we will keep all functions returning the same data structure. We had some discussion about the same situation at #23. In fact, the #23 aims to change the `join` pipe to follow this approach :+1:.

**Review by akshayphilar — APPROVED** [review]

**Review by renancunha — APPROVED** [review]

**Comment by VMRuiz** [comment]
> Great feature  @VMRuiz! During the schema conversion tests, we needed a feat like this one to fill a specific field with a fixed value.
> 
> I have just one consideration: maybe should we stick to the idea of always returns the iterable (with the result), instead of returning the result itself? By doing this we will keep all functions returning the same data structure. We had some discussion about the same situation at #23. In fact, the #23 aims to change the `join` pipe to follow this approach .

Hello @renancunha , 

I thought about this and I decided to allow the user to get either a single element `identity(1)` or a iterable  `identity((1,))` because there are others method that doesn't return iterables like `join` , `first`, and all of the string methods. 

However, I don't have anything agaisn't change it if you all think is better.

File: shublang/shublang.py [hunk 1]
```diff
@@ -332,6 +332,11 @@ def extract_currency(iterable):
 def urljoin(iterable, base):
     return (parse.urljoin(base, url) for url in iterable)
 
+@Pipe
+def identity(iterable, element):
+    """ Return the same element is passed as parameter."""
+    return element
```
Comment (akshayphilar) [inline shublang/shublang.py:338]:
```suggestion
    return (element)
```

Agree with @renancunha, possibly we can return an iterable by default?

Comment (akshayphilar) [reply shublang/shublang.py:338]: While the user can flatten this with `first`, the universal parser also does this by design. That way we can stick to the pipe philosophy

**Pull request merged**